server/src/main/java/org/opensearch/index/fielddata/fieldcomparator/FloatValuesComparatorSource.java

```diff
@@ -78,7 +78,7 @@ public SortField.Type reducedType() {
         return SortField.Type.FLOAT;
     }
 
-    private NumericDoubleValues getNumericDocValues(LeafReaderContext context, float missingValue) throws IOException {
+    protected NumericDoubleValues getNumericDocValues(LeafReaderContext context, float missingValue) throws IOException {
```

Comment:
Do you think we should better off copy this method to `HalfFloatXxx` counterpart instead of opening it up?

Reply:
+1 , that would be better, done.